src/main/java/com/spikes2212/command/drivetrains/commands/DriveTank.java

```diff
@@ -17,41 +17,123 @@ public class DriveTank extends CommandBase {
     protected final Supplier<Double> leftSpeedSupplier;
     protected final Supplier<Double> rightSpeedSupplier;
     protected final Supplier<Boolean> isFinished;
+    protected final boolean squareInputs;
 
     /**
      * This constructs a new {@link DriveTank} command that moves the given
      * {@link TankDrivetrain} according to speed values from Double {@link Supplier}s for left and right sides.<br>
-     * Positive values move forwards.
+     * Positive values move the drivetrain forward.
      *
      * @param drivetrain         the drivetrain this command requires and moves.
-     * @param leftSpeedSupplier  the double {@link Supplier} supplying the speed to move on the left side with.
-     * @param rightSpeedSupplier the double {@link Supplier} supplying the speed to move on the right side with.
+     * @param leftSpeedSupplier  the double {@link Supplier} supplying the speed to move the left side with,
+     *                           ranging from -1 to 1.
+     * @param rightSpeedSupplier the double {@link Supplier} supplying the speed to move the right side with,
```

Comment:
some constructor docs use a dot rather than a comma here, decide which one should be used for all of them